argo_workflows/metadata.csv

```diff
@@ -0,0 +1,46 @@
+argo_workflows.go.memstats.heap.sys_bytes,gauge,,byte,,Number of heap bytes obtained from system.,0,argo_workflows,,
+argo_workflows.go.memstats.last_gc_time_seconds,gauge,,second,,Number of seconds since 1970 of last garbage collection.,0,argo_workflows,,
+argo_workflows.go.memstats.lookups.count,count,,,,Total number of pointer lookups.,0,argo_workflows,,
+argo_workflows.go.memstats.mallocs.count,count,,,,Total number of mallocs.,0,argo_workflows,,
+argo_workflows.go.memstats.mcache.inuse_bytes,gauge,,byte,,Number of bytes in use by mcache structures.,0,argo_workflows,,
+argo_workflows.go.memstats.mcache.sys_bytes,gauge,,byte,,Number of bytes used for mcache structures obtained from system.,0,argo_workflows,,
+argo_workflows.go.memstats.mspan.inuse_bytes,gauge,,byte,,Number of bytes in use by mspan structures.,0,argo_workflows,,
+argo_workflows.go.memstats.mspan.sys_bytes,gauge,,byte,,Number of bytes used for mspan structures obtained from system.,0,argo_workflows,,
+argo_workflows.go.memstats.next.gc_bytes,gauge,,byte,,Number of heap bytes when next garbage collection will take place.,0,argo_workflows,,
+argo_workflows.go.memstats.other.sys_bytes,gauge,,byte,,Number of bytes used for other system allocations.,0,argo_workflows,,
+argo_workflows.go.memstats.stack.inuse_bytes,gauge,,byte,,Number of bytes in use by the stack allocator.,0,argo_workflows,,
+argo_workflows.go.memstats.stack.sys_bytes,gauge,,byte,,Number of bytes obtained from system for stack allocator.,0,argo_workflows,,
+argo_workflows.go.memstats.sys_bytes,gauge,,byte,,Number of bytes obtained from system.,0,argo_workflows,,
+argo_workflows.go.threads,gauge,,,,Number of OS threads created.,0,argo_workflows,,
+argo_workflows.go.info,gauge,,,,Information about the Go environment.,0,argo_workflows,,
+argo_workflows.k8s_request.count,count,,request,,"Number of kubernetes requests executed. https://argo-workflows.readthedocs.io/en/release-3.5/metrics/#argo_workflows_k8s_request_total",0,argo_workflows,,
+argo_workflows.log_messages.count,count,,message,,Total number of log messages.,0,argo_workflows,,
+argo_workflows.pods,gauge,,,,Number of Pods from Workflows currently accessible by the controller by status (refreshed every 15s),0,argo_workflows,,
+argo_workflows.queue_adds.count,count,,,,Adds to the queue,0,argo_workflows,,
+argo_workflows.queue_depth,gauge,,,,Depth of the queue,0,argo_workflows,,
+argo_workflows.workers_busy,gauge,,worker,,Number of workers currently busy,0,argo_workflows,,
+argo_workflows.workflow_condition,gauge,,,,"Workflow condition. https://argo-workflows.readthedocs.io/en/release-3.5/metrics/#argo_workflows_workflow_condition",0,argo_workflows,,
+argo_workflows.workflows_processed.count,count,,,,Number of workflow updates processed,0,argo_workflows,,
+argo_workflows.operation_duration_seconds.bucket,count,,,,The count of observations in the histogram of durations of operations split into buckets by upper bound.,0,argo_workflows,,
```

Comment:
Should we add the `second` unit?

Reply:
Histograms are tricky: .bucket and .count track **number of samples**, not the samples themselves. So it's meta 🙄